test/cli/package-test-simple/test.out

```diff
@@ -1,13 +1,8 @@
-main_lib/lib.rb:8: No import provides `Project::TestOnly` https://srb.help/5002
+main_lib/lib.rb:8: Used `test_import` constant `Project::TestOnly::SomeHelper` in non-test file https://srb.help/3720
```

Comment:
Do you mind starting a list of follow-up items that would be good to get to eventually?

(Was `test_import` one of the things we were going to delete? Or was it `export_for_test`? Any others?)

Happy for the list to live either

- as a bullet list somewhere on this PR
- as github issues
- as Stripe-internal Jira tickets
- as a Stripe-internal document

(at your preference)